### PR TITLE
Role binding for RBAC authorization

### LINKED ISSUE #10: RBAC for non-legacy authorization

### PR DESCRIPTION
**Pull request by grantr**
Adds a ClusterRoleBinding that gives the default service account in the default namespace the `cluster-admin` role. That replicates the authorization situation in pre-RBAC clusters. It's not a long-term fix, but it will unblock development on 1.8 clusters.

`kubectl create -f artifacts/examples/role-binding.yaml`

Fixes #10

### CODE REVIEW

File: artifacts/examples/role-binding.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
```
Comment (mattmoor) [inline artifacts/examples/role-binding.yaml]: You don't need this in a single-object file.

**Comment by grantr** [comment]
Note this gives every pod in the `default` namespace admin rights. A better solution would be to create a new role as in https://github.com/fabric8io/fabric8/issues/6840#issuecomment-322984003, something like this (but I don't know if this one actually works):

```yaml
---
apiVersion: rbac.authorization.k8s.io/v1beta1
kind: Role
metadata:
  name: foo-controller
  namespace: default
rules:
- apiGroups: [""]
  resources: ["foos"]
  verbs: ["get", "watch", "list"]
```

**Comment by grantr** [comment]
@mattmoor RFAL

**Comment by mattmoor** [comment]
I assume you confirmed this works? :D

**Comment by grantr** [comment]
I did indeed :smiley:

**Comment by grantr** [comment]
This seems to have broken things for @mattmoor, possibly because he's running kubectl from a VM and I'm on a workstation. Seems like the blocker is giving the current kubectl user permissions to create RBAC resources.

**Pull request merged**